src/program/lwaftr/query/README

```diff
@@ -0,0 +1,16 @@
+Usage:
+  query [OPTIONS] [<pid>]
```

Comment:
The code allows to filter the shown counters by name (very nice, btw): the usage description has to mention that too.